boards/adafruit-trinkey-qt2040/Cargo.toml

```diff
@@ -0,0 +1,29 @@
+[package]
+name = "adafruit-trinkey-qt2040"
+version = "0.1.0"
+authors = ["Dylan Ede <dylanede@googlemail.com>"]
```

Comment:
Would you mind setting this to "The rp-rs Developers"? I'm trying to get them all changed to be the same for consistency.

[See https://github.com/rp-rs/rp-hal/pull/245](https://github.com/rp-rs/rp-hal/pull/245#issuecomment-1000231052)

Reply:
Sure